docs/storage.md

```diff
@@ -129,3 +129,24 @@ Note that on the read path, Prometheus only fetches raw series data for a set of
 ### Existing integrations
 
 To learn more about existing integrations with remote storage systems, see the [Integrations documentation](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage).
+
+## Backfilling from OpenMetrics format
+
+### Overview
+
+If a user wants to migrate from an existing monitoring system (source) to Prometheus, they can add their older data to TSDB easily using backfilling. The source system may be another TSDB which is capable of dumping data in [OpenMetrics](https://openmetrics.io/) format. However,one should be careful and note that it is not safe to backfill data from the last 3 hours (the current head block).
```

Comment:
This seems to be written the wrong way around. The purpose of this is creating blocks that come from the OM format (which could come from another monitoring system as part of a migration). Not that this is a way to migrate from an existing monitoring system (that happens to involve OM and creating blocks), as that's only one potential use case and would cause confusion about what this actually is.

Focus primarily what this is, not what it could be used for.

Reply:
done